### PR TITLE
fix: Fixed compatibility with SlikSVN

### PR DESCRIPTION
**Pull request by edgardmessias**

### CODE REVIEW

File: src/svnFinder.ts [hunk 1]
```diff
@@ -112,9 +112,11 @@ export class SvnFinder {
 
   public checkSvnVersion(svn: ISvn): Promise<ISvn> {
     return new Promise<ISvn>((c, e) => {
-      if (!semver.valid(svn.version)) {
+      // fix compatibility with SlickSVN (like 1.6.17-SlikSvn-tag-1.6.17@1130898-X64)
+      const version = svn.version.replace(/^(\d+\.\d+\.\d+).*/, "$1");
```
Comment (JohnstonCode) [inline src/svnFinder.ts]: whats the `.*` needed for? `^(\d+\.\d+\.\d+)` gets the version alone

Comment (edgardmessias) [reply src/svnFinder.ts]: Fast version instead of match or exec

**Comment by edgardmessias** [comment]
I use replace for single code:
With exec:
```js
...
let version = svn.version;
const match = /^(\d+\.\d+\.\d+)/.exec(svn.version);
if (match && match[0]) {
    version = match[0]
}
...
```

**Pull request merged**